debug-on-kubernetes.md

```diff
@@ -0,0 +1,36 @@
+# Debug support on Kubernetes cluster
+
+## 1. Supported languages
+   * `java` (Required: [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) extension)
+   * `node`
+
+## 2. Commands for debugging
+   * `Kubernetes: Debug (Launch)` - Run the current application as a Kubernetes Deployment and attach a debugging session to it (currently works only for Java/Node.js deployments)
+   * `Kubernetes: Debug (Attach)` - Attach a debugging session to an existing Kubernetes Deployment (currently works only for Java deployments)
+
+## 3. How to use it for java debugging
+### 3.1 Launch a spring-boot application on Kubernetes and debug it
+   * Launch VSCode
+   * Open a spring-boot application
+   * Containerize first. (if Dockerfile already exists, skip it)
+   * Install [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) extension (if not)
+   * Login in docker client with your Docker Hub or other registries (e.g. Azure ACR) account, and set up the image repository url on the VSCode User Settings (if debugging on minikube, skip it)

```

Comment:
I'd suggest flipping this and splitting it in two:

* Set up the image repository in the VS Code user or workspace settings, using the `vsdocker.imageUser` configuration setting.  (If debugging on minikube, skip this step.)

(...example config...)

* Log into your repository using the appropriate tool.  For example, to log into Docker Hub, run `docker login`; to log into Azure Container Registry, run `az acr login`.